aea/helpers/acn/agent_record.py

```diff
@@ -119,18 +139,29 @@ def message(self) -> bytes:
         return self._message
 
     @property
-    def ledger_id(self) -> str:
+    def identifier(self) -> SimpleIdOrStr:
+        """Get the identifier."""
+        return self._identifier
```

Comment:
if the above is accepted:
```suggestion
        return str(self._identifier)
```

Reply:
but with the second alternative `str(SimpleId(...))` then not needed